pennylane_lightning_kokkos/src/simulator/StateVectorKokkos.hpp

```diff
@@ -30,6 +31,9 @@
 #include "GateFunctors.hpp"
 #include "MeasuresFunctors.hpp"
 
+std::string repr_InitArguments(const Kokkos::InitArguments &a);
+void print_InitArguments(const Kokkos::InitArguments &a);
```

Comment:
It is preferable not to use forward declarations anywhere we can avoid them --- since no other part of Lightning Kokkos is included in this way, we should follow the same structure of the existing code. Is there another design that can be used to ensure the new functions work as a header templated library?

Reply:
Ok, thanks for the explanations. I put the definitions in `pennylane_lightning_kokkos/src/util/KokkosUtil.hpp`. So one can include `KokkosUtil.hpp` to get the streaming operator for `Kokkos::InitArguments` structs.